packages/e2e-tests/register/models/SetRecoveryOption.js

```diff
@@ -0,0 +1,20 @@
+class SetRecoveryOptionPage {
+    constructor(page) {
+        this.page = page;
+    }
+    async navigate(accountId) {
+        await this.page.goto(`/set-recovery/${accountId}`);
+    }
+    async selectRecoveryOption(searchText) {
+        await this.page.click(`div :text-matches("${searchText}", "i")`);
+    }
+    async submitRecoveryOption(searchText) {
+        await this.selectRecoveryOption(searchText);
+        await this.page.click(`[type="submit"]`);
+    }
+    // optionName is option prop in code (currently /phrase|ledger|email|phone/)
```

Comment:
the option prop is what's currently used in the frontend code [here](https://github.com/near/near-wallet/blob/account-creation-e2e-tests/packages/frontend/src/components/accounts/recovery_setup/RecoveryOption.js#L215). I setup the testIds so that they are based on that prop just like icons and active state etc.

The way it's written currently, the element that has the `active` prop we need to make assertions on does not trigger the active state. So to circumvent this we use the test-id to make assertions on the `active` prop and the searchText to toggle it by clicking.